packages/belos/src/BelosStatusTestGenResSubNorm.hpp

```diff
@@ -93,7 +93,7 @@ class StatusTestGenResSubNorm: public StatusTestResNorm<ScalarType,MV,OP> {
     @param showMaxResNormOnly: for output only
 
   */
-  StatusTestGenResSubNorm( MagnitudeType Tolerance, size_t subIdx, int quorum = -1, bool showMaxResNormOnly = false ) {
+  StatusTestGenResSubNorm( MagnitudeType /* Tolerance */, size_t /* subIdx */, int /* quorum = -1 */, bool /* showMaxResNormOnly = false */ ) {
```

Comment:
You should actually keep the default values of parameters -- otherwise, you might break code that only gives 2 or 3 arguments to this constructor.  For example:
```c++
StatusTestGenResSubNorm( MagnitudeType /* Tolerance */, size_t /* subIdx */, int /* quorum */ = -1, bool /* showMaxResNormOnly */ = false ) {
... 
}
```